### ISSUE TITLE
New colors should be added for accessibility

### ISSUE THREAD

**Issue by codepretty** [microsoft/fluent-ui-react#763]
## Bug Report
An accessibility issue was found with at-mentioning someone in a me chat bubble. The color contrast ratio isn't correct.

### Fix
Add 2 new colors and update the background color and at-mention text color in the chat message me bubble.

![image](https://user-images.githubusercontent.com/828692/51634649-a088d700-1f09-11e9-96d7-cdcaff440432.png)

**Add new colors**
brand05: #585a96
brand15: #e5e5f1

**Update the chat me message**
The link changes to brand05
The me bubble should changes to: brand15

**Comment by mnajdova** [comment]
@codepretty was this resolved? Can we close this one?

**Comment by codepretty** [comment]
I think the chatMessageVariables need to be updated `linkColorMine: siteVars.colorScheme.brand.foreground2` needs to be updated to `foreground3` based on the link colors below

![image](https://user-images.githubusercontent.com/828692/58047472-25bd7a80-7afd-11e9-91c4-b2da4c299bf5.png)

**Comment by alinais** [comment]
Closing. Both of these hex values are now accounted for in the Stardust palette as well as implemented in Angular
